src/contracts/GPv2AccessControl.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+import "./interfaces/GPv2Authentication.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AccessControl is Ownable, GPv2Authentication {
```

Comment:
naming-nit: I feel like `AccessControl` doesn't relate very well to the interface its implementing. What do you think about:
```suggestion
contract GPv2SimpleAuthentication is Ownable, GPv2Authentication {
```

Reply:
Yes, will do!